doc/devel/release_guide.rst

```diff
@@ -176,14 +176,12 @@ Finally, push the tag to GitHub::
 
 Congratulations, the scariest part is done!
 
-.. [#] The tarball that is provided by GitHub is produced using `git
-       archive <https://git-scm.com/docs/git-archive>`__.  We use
-       `versioneer <https://github.com/warner/python-versioneer>`__
-       which uses a format string in
+.. [#] The tarball that is provided by GitHub is produced using `git archive`_.
+       We use setuptools_scm_ which uses a format string in
```

Comment:
```suggestion
       We use `setuptools_scm`_ which uses a format string in
```

Reply:
A quick test shows this isn't needed in rst?

Reply:
:ballot_box_with_check: I've learned something new.

While the Sphinx [reStructuredText Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links) only mentions the variant with backticks, the [specification](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#hyperlink-references) indeed allows to leave them out. (Could somebody donate a tiny bit of CSS to the docutils folks? - The no-CSS style of the specs is quite unreadable, which is why I try to avoid going there.)